conda-store-server/conda_store_server/_internal/worker/tasks.py

```diff
@@ -298,11 +298,41 @@ def task_delete_build(self, build_id):
 
         # Updates build size and marks build as deleted
         build.deleted_on = datetime.datetime.utcnow()
+        build.status = schema.BuildStatus.DELETED
         build.size = 0
 
         db.commit()
 
 
+@shared_task(base=WorkerTask, name="task_archive_build", bind=True)
+def task_archive_build(self, build_id):
+    conda_store = self.worker.conda_store
+    with conda_store.session_factory() as db:
+        build = api.get_build(db, build_id)
+
+        archive_save_artifacts = [
```

Comment:
there is a setting to allow users to configure what artifacts should be deleted when a build is deleted. Should we add a setting that allows users to customize what artifacts are deleted on archive as well?